### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by Dablup**
First lab completed

### CODE REVIEW

File: app_python/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Time show web application
+
+## Overview
+The is simple application that show current MSK time.
```
Comment (SMore-Napi) [inline app_python/README.md]: shows*

**Review by al1ych — APPROVED** [review]
lgtm)

**Comment by emileyray** [comment]
Nice

**Review by MikhailMarch — COMMENTED** [review]
Nice

**Pull request closed without merging**